src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/RolesMappingApiTest.java

```diff
@@ -392,4 +392,16 @@ public void testRolesMappingApiForNonSuperAdmin() throws Exception {
 		Assert.assertEquals(HttpStatus.SC_NOT_FOUND, response.getStatusCode());
 
 	}
+
+	@Test
+	public void checkNullElementsInArray() throws Exception{
+		setup();
+		rh.keystore = "restapi/kirk-keystore.jks";
+		rh.sendAdminCertificate = true;
+
+		HttpResponse response = rh.executePutRequest("/_opendistro/_security/api/rolesmapping/opendistro_security_role_starfleet_captains",
+													 FileHelper.loadFile("restapi/rolesmapping_null_array_element.json"), new Header[0]);
+		System.out.println(response.getBody());
+		Assert.assertTrue(response.getBody().contains("Invalid array element"));
```

Comment:
nit: Can we assert for `SC_BAD_REQUEST` here ? 
Also would be good to add tests for other APIs accepting array type

Reply:
Sure, I changed to `SC_BAD_REQUEST` and also added unit tests for other APIs.